orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/PipelineExpressionEvaluator.java

```diff
@@ -82,13 +86,15 @@ public PipelineExpressionEvaluator(List<ExpressionFunctionProvider> expressionFu
   private final Function<String, String> includeExecutionParameter =
       e -> {
         String expression = e;
-        for (String fn : EXECUTION_AWARE_FUNCTIONS) {
+        for (String fn : this.executionAwareFunctions) {
           if (expression.contains("#" + fn)
               && !expression.contains("#" + fn + "( #root.execution, ")) {
             expression = expression.replaceAll("#" + fn + "\\(", "#" + fn + "( #root.execution, ");
           }
         }
 
+        // 'deployServerGroups' is a variable instead of a function and this block handles that.
+        // Migrate it to a function before removing this block of code.
```

Comment:
it's really that one must migrate the old pipelines that use it as a `variable`

Reply:
Corrected the comment line.